app/selenium_ui/jsd/pages/agent_pages.py

```diff
@@ -46,3 +47,13 @@ class BrowseProjects(BasePage):
 
     def wait_for_page_loaded(self):
         self.wait_until_visible(BrowseProjectsLocators.page_title)
+
+
+class BrowseCustomers(BasePage):
+
+    def __init__(self, driver, project_key=None):
+        BasePage.__init__(self, driver)
+        self.page_url = UrlManager(project_key=project_key).browse_project_customers_page_url()
```

Comment:
Let's follow code style convention for page objects as we have in already developed solutions for Jira, etc.
`self.url_manager = UrlManager(project_key=project_key), self.url_manager.browse_project_customers_page_url()`

Reply:
Just url_manager not self.url_manager ?